pyvista/utilities/helpers.py

```diff
@@ -437,13 +437,62 @@ def vector_poly_data(orig, vec):
     return pyvista.PolyData(pdata)
 
 
-def trans_from_matrix(matrix):
-    """Convert a vtk matrix to a numpy.ndarray."""
-    t = np.zeros((4, 4))
-    for i in range(4):
-        for j in range(4):
-            t[i, j] = matrix.GetElement(i, j)
-    return t
+def trans_from_matrix(matrix):  # pragma: no cover
+    """Convert a vtk matrix to a numpy.ndarray.
+
+    DEPRECATED: Please use ``array_from_vtkmatrix``.
+
+    """
+    raise DeprecationError('DEPRECATED: Please use ``array_from_vtkmatrix``.')
```

Comment:
We need to import `DeprecationError`:
```suggestion
    from pyvista.core.errors import DeprecationError
    raise DeprecationError('DEPRECATED: Please use ``array_from_vtkmatrix``.')
```

Reply:
Bah, I forgot to run the deprecated function. Sorry.